db/migrate/20170703170753_rename_applies_to_exp_to_miq_expression_visible.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAppliesToExpToMiqExpressionVisible < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :custom_buttons, :applies_to_exp, :miq_expression_visible
```

Comment:
`visible` in an adjective, so this reads funny to me.  I'm thinking `visibility_expression` or `visibility_miq_expression`?  Thoughts?  Similar discussion in #29